lab-beeksma/test/test.js

```diff
@@ -0,0 +1,61 @@
+const assert = require('assert');
+const greet = require('../lib/greet.js');
+const arithmetic = require ('../lib/arithmetic.js');
+
+
+describe('greet', function() {
+  describe('#greet()', function() {
+
+    it('should return "Hello World!"', function() {
+      assert.strictEqual(greet.greet('World!'), 'Hello World!');
+    });
+
+    it('should return NULL if an argument that is not a string is passed' , function() {
+      assert.strictEqual(greet.greet(true), null);
+    });
+
+  });
+});
+
+describe('arithmetic', function() {
+
+  describe('#add()', function() {
+
+    it('should return 17 when passed 8 and 9', function() {
+      assert.strictEqual(arithmetic.add(8,9), 17);
+    });
+
+    it('should return 8 when passed 8 and null', function() {
+      assert.strictEqual(arithmetic.add(8,null), 8);
```

Comment:
This is an example of a test where I'd include another `assert` with the parameters flipped. Low cost to maintain, nonzero chance of messing up the related logic.